sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/AccountDataResolver.scala

```diff
@@ -0,0 +1,5 @@
+package com.azure.cosmos.spark
```

Comment:
please add license header similar to other files.

Reply:
Resoved